extensions/amp-video/1.0/storybook/VideoIframe.js

```diff
@@ -62,6 +62,7 @@ const AmpVideoIframeLike = ({unloadOnPause, ...rest}) => {
       allow="autoplay" // this is not safe for a generic frame
       onMessage={onMessage}
       makeMethodMessage={makeMethodMessage}
+      data-loading="lazy"
```

Comment:
Is this necessary? Should we set the default at the `<VideoIframe />` level instead?

Reply:
Removed. Remnant of testing.